docs/advanced-test-features/using-test-kits-in-ci-cd.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Using Test Kits in CI/CD
+nav_order: 14
+parent: Advanced Features
```

Comment:
This definitely needs to be under the 'Using Test Kits' section, not the 'Writing Test Kits'.  You have "Using Test Kits ..." in the name of this section!

Reply:
Since I would like to link to the docs in tomorrow's Tech Talk, I'm going ahead and making this change.

Reply:
I updated this to go here:

<img width="851" alt="Screenshot 2024-11-12 at 9 27 57 PM" src="https://github.com/user-attachments/assets/c925d97e-1c51-4673-b2ce-88a2c0f5b5ea">